terraform/main.tf

```diff
@@ -0,0 +1,96 @@
+# INSTRUCTIONS: 
+# 1) ENSURE YOU POPULATE THE LOCALS 
+# 2) ENSURE YOU REPLACE ALL INPUT PARAMETERS, THAT CURRENTLY STATE 'ENTER VALUE', WITH VALID VALUES 
+# 3) YOUR CODE WOULD NOT COMPILE IF STEP NUMBER 2 IS NOT PERFORMED!
+# 4) ENSURE YOU CREATE A BUCKET FOR YOUR STATE FILE AND YOU ADD THE NAME BELOW - MAINTAINING THE STATE OF THE INFRASTRUCTURE YOU CREATE IS ESSENTIAL - FOR APIS, THE BUCKETS ALREADY EXIST
+# 5) THE VALUES OF THE COMMON COMPONENTS THAT YOU WILL NEED ARE PROVIDED IN THE COMMENTS
+# 6) IF ADDITIONAL RESOURCES ARE REQUIRED BY YOUR API, ADD THEM TO THIS FILE
+# 7) ENSURE THIS FILE IS PLACED WITHIN A 'terraform' FOLDER LOCATED AT THE ROOT PROJECT DIRECTORY
+
+provider "aws" {
+  region  = "eu-west-2"
+  version = "~> 2.0"
+}
+data "aws_caller_identity" "current" {}
+data "aws_region" "current" {}
+locals {
+   //application_name = your application name # The name to use for your application
```

Comment:
Do we need this? If not, let's clean it up

Reply:
We do if we go with Fargate, so I will leave it for now until it is decided that we need to remove the Fargate set up

Reply:
👍